waves-integration/src/test/scala/com/wavesplatform/dex/grpc/integration/caches/BlockchainCacheSpecification.scala

```diff
@@ -52,16 +53,14 @@ class BlockchainCacheSpecification extends WavesIntegrationSuiteBase with Before
 
       val badKeyAccessCount = 10
 
-      Await.result(
-        (1 to badKeyAccessCount).foldLeft(Future.successful("")) { (prev, _) =>
-          for {
-            _ <- prev
-            _ <- cache get goodKey
-            r <- cache get badKey recover { case _ => "sad" }
-          } yield { Thread.sleep(andThenAwaitTimeout); r }
-        },
-        scala.concurrent.duration.Duration.Inf
-      )
+      val future = (1 to badKeyAccessCount).foldLeft(Future.successful("")) { (prev, _) =>
+        for {
+          _ <- prev
+          _ <- cache get goodKey
+          r <- cache get badKey recover { case _ => "sad" }
+        } yield { Thread.sleep(andThenAwaitTimeout); r }
+      }
+      future.isReadyWithin(10.minutes) shouldBe true
```

Comment:
1. Please use `futureValue` without arguments. The explanation is above.
2. I think, we could move `Thread.sleep` after `futureValue`